### PR TITLE
update doc to make sure mail is routed both ways

### PR DESCRIPTION
**Pull request by samschan-ms**
@KCCross  please review

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ If you recently migrated from AD RMS to Azure Information Protection, you'll nee
...
+> For hybrid Exchange environments, on-premises users can send and receive encrypted mail using OME only if email is routed through Exchange Online. To configure OME in a hybrid Exchange environment, you need to first [configure hybrid using the Hybrid Configuration wizard](https://docs.microsoft.com/Exchange/exchange-hybrid) and then [configure mail to flow between your email server and Office 365](https://docs.microsoft.com/en-us/exchange/mail-flow-best-practices/use-connectors-to-configure-mail-flow/set-up-connectors-to-route-mail#part-2-configure-mail-to-flow-from-your-email-server-to-office-365). Once you've configured mail to flow through Office 365, then you can configure mail flow rules for OME by using this guidance.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md]:
```suggestion
> For hybrid Exchange environments, on-premises users can send and receive encrypted mail using OME only if email is routed through Exchange Online. To configure OME in a hybrid Exchange environment, you need to first [configure hybrid using the Hybrid Configuration wizard](https://docs.microsoft.com/Exchange/exchange-hybrid) and then [configure mail to flow between your email server and Office 365](https://docs.microsoft.com/exchange/mail-flow-best-practices/use-connectors-to-configure-mail-flow/set-up-connectors-to-route-mail#part-2-configure-mail-to-flow-from-your-email-server-to-office-365). Once you've configured mail to flow through Office 365, you can configure mail flow rules for OME by using this guidance.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove the locale from the link and remove the word "then" from the last sentence.

**Comment by mypil** [comment]
@samschan-ms - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Comment by samschan-ms** [comment]
@KCCross  I've added one more link to setup incoming mail flow.

**Pull request merged**